paasta_tools/autoscaling/autoscaling_cluster_lib.py

```diff
@@ -73,16 +91,26 @@ def log(self):
 
 class ClusterAutoscaler(ResourceLogMixin):
 
-    def __init__(self, resource, pool_settings, config_folder, dry_run, log_level=None, draining_enabled=True):
+    def __init__(
+        self,
+        resource,
+        pool_settings,
+        config_folder,
+        dry_run,
+        utilization_error,
+        log_level=None,
+        draining_enabled=True,
+    ):
         self.resource = resource
         self.pool_settings = pool_settings
         self.config_folder = config_folder
         self.dry_run = dry_run
+        self.utilization_error = utilization_error
         self.draining_enabled = draining_enabled
         if log_level is not None:
             self.log.setLevel(log_level)
 
-    def set_capacity(self, capacity):
+    def set_capacity(self, capacity, unrounded_capacity):
```

Comment:
Would much of this code be simpler if this *only* took and then relied on the implementation to do the rounding at the last minute?